frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTable.tests.tsx

```diff
@@ -193,26 +269,35 @@ describe("QueriesTable", () => {
       "Team query 2",
     ];
 
-    testData.forEach(async (tableProps) => {
-      // will have no context to get current user from
-      const { user } = renderAsPremiumGlobalAdmin(
-        <QueriesTable {...tableProps} />
-      );
-      dataStrings.forEach((val) => {
-        expect(screen.getAllByText(val)[0]).toBeInTheDocument();
-      });
-
-      await user.type(
-        screen.getByPlaceholderText("Search by name"),
-        "shouldn't match anything"
-      );
-      expect(screen.getByText("No matching queries")).toBeInTheDocument();
-      dataStrings.forEach((val) => {
-        expect(screen.getAllByText(val)).toHaveLength(0);
-      });
```

Comment:
FYI `.forEach` doesn't work with async, so this wasn't actually testing anything.  We'd want to do `for...in` for that to work.  In any case, changing the search value in the table wouldn't cause the changes expected here because those are handled by the ManageQueriesPage component.  I split this into separate tests.